pywbemtools/pywbemcli/_common.py

```diff
@@ -1599,7 +1593,7 @@ def all_classnames_depsorted(namespace, conn):
         dep_classnames = dependent_classnames(cls)
         all_deps[cls.classname] = set(dep_classnames)
 
-    # TODO: Add support for case insensitive sorting.
+    # ISSUE $ 954: Add support for case insensitive sorting.
```

Comment:
In other places, we use the notation `#NNN`, and I suggest to be consistent eg for searching.

Reply:
Fixed